Automattic/newspack-blocks
# Register Newspack Patterns like Core instead of custom solution

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates the way we offer our Patterns to users. Previously we used a custom solution. Now we can rely on Core's functions to offer them.

_Note: The previews for the Homepage Posts are quite slow (at least on local). Not quite sure what we can do about it._

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Test all the different patterns. Do you see them, do they work?

### Other information:
